java/scala-logback/README.md

```diff
@@ -0,0 +1,9 @@
+# Sentry Scala + Logback Example
+
+This project contains an integration example written in Scala and uses Sentry's Logback SDK.
+
+## Installation and Configuration
+1. Add the Sentry Logback package as dependency to the [build.sbt](https://github.com/getsentry/examples/blob/scala-logback/java/scala-logback/build.sbt#L8) file.
```

Comment:
I think you directories are out of order here: `blob/scala-logback/java/scala-logback`

But also while fixing that, I think maybe this example should just live at `$PROJECT_ROOT/scala/logback` since it's about Scala and not Java. Mostly so users can easily find it since they'll see it's broken out by language at the toplevel and expect to see Scala there I think?

Reply:
Bumping this one since GH collapsed it.

Reply:
I've moved the example into a `/scala/logback` and made the appropriate edits to the README.

Reply:
Still need `scala-logback` -> `master` in the URL or else it will point to the branch (which may be deleted or outdated) once we merge.